### PR TITLE
resolves #133 : added profileImage attr in curl cmd under Update User

### LINKED ISSUE #133: [Doc]  modifie our  profile image
UPDATE : 
replace on server Documentation in the update user line 
- curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo"}'

by :
-  curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo"**, profileImage : "link to profile image"**}'

### PR DESCRIPTION
**Pull request by mpsb**
Hello @Benvii , thank you for the help! Please check if this is okay. Made this a draft PR just in case.

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ curl http://localhost:5000/api/user -H "Content-Type: application/json" --data '
 
 Update User:
 ```sh
-curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo"}'
+curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo", "profileImage": "link to profile image"}'
```
Comment (Benvii) [inline server/README.md:131]: Tested but doesn't work, there is an error in the field name on the ticket, it's called `profilImage` (on the server) not `profileImage` ([see model](https://github.com/mdl29/scratchy/blob/main/server/scratchy_server/model/userModel.py#L6)).

Can you update it please ? Thanks in advance for your help.

**Review by Benvii — CHANGES_REQUESTED** [review]

**Comment by Draeggon** [comment]
it will be change to profileImage in #120

**Comment by Benvii** [comment]
> it will be change to profileImage in #120

Clearly it shouldn't be on that PR as it has nothing linked with the subject of the PR. Well I will review it, but it best to make small PR that only address their respective tickets otherwise stuff are blocked like in this case, as #120 is complex to review.

**Comment by mpsb** [comment]
Hello all. In that case, should I change the command under `Create User:` too to keep it consistent? The JSON attribute of that command is also `profileImage`:
```
curl http://localhost:5000/api/user -H "Content-Type: application/json" --data '{"pseudo": "mon pseudo", 
"profileImage": "https://http.cat/204"}'
```

Also, I checked for occurrences of `profilImage`, and it only shows in `server/README.md` and `userModel.py`. As for occurrences of `profileImage`, it's actually used in `server/scratchy_server/resources/userres.py` for this class method:
```
def post(self):
        userData = request.get_json()
        user = UserModel()
        # room.id = uuid.uuid4().hex
        user.pseudo = userData['pseudo'] if 'pseudo' in userData else "unknow pseudo"
        user.profileImage = userData['profileImage'] if 'profileImage' in userData else "default profile image"
        user.user = userData['user'] if 'user' in userData else "default"

        try:
            user = user.save()
        except NotUniqueError:
            logging.debug("the pseudo is already used")
            abort(409)
        logging.debug("user: '%s' has been created", user.user)
        return {'id': str(user.id)}
```

Maybe I should just change the `profilImage` attribute in `server/scratchy_server/model/userModel.py`to `profileImage`?

**Comment by Benvii** [comment]
Hi @mpsb correct, you can also refactor those occurrences. Thanks in advance.

File: server/README.md (hunk 1 above)
Comment (mpsb) [reply server/README.md:131]: Refactored profileImage attribute of UserModel instead.

**Review by Draeggon — DISMISSED** [review]

**Review by mpsb — COMMENTED** [review]
Opted to change the attribute name in `server/scratchy_server/model/userModel.py` instead.

**Review by Benvii — DISMISSED** [review]
It's all right, thanks for your contribution :+1:

**Review by Benvii — APPROVED** [review]

**Pull request merged**